tools/docker/docker-entrypoint.sh

```diff
@@ -54,6 +54,17 @@ else
   echo "[INFO] No custom assets found at ${CUSTOM_ASSETS_PATH}"
 fi
 
+# Executing custom scripts located in CUSTOM_SCRIPTS_DIRECTORY if environment variable is set
+if [[ -z "${CUSTOM_SCRIPTS_DIRECTORY}" ]]; then
+  echo "[INFO] No CUSTOM_SCRIPTS_DIRECTORY env variable set"
+else
+  echo "[INFO] CUSTOM_SCRIPTS_DIRECTORY env variable set to ${CUSTOM_SCRIPTS_DIRECTORY}"
+  chmod +x ${CUSTOM_SCRIPTS_DIRECTORY}/*.sh  
+  cd ${CUSTOM_SCRIPTS_DIRECTORY}/
```

Comment:
I'm not sure if this line is really necessary; it works with it because the directory is an absolute path, but it could be misleading maybe?

Reply:
I don't think it's necessary either